tasks/config.yml

```
@@ -53,9 +53,7 @@
      --consensusCommon-suggestedBlockGasLimit={{ rocketpool_consensus_layer_block_gas_limit }} \
      --native-ecHttpUrl={{ rocketpool_eth1_exec_layer_rpc_url | mandatory }} \
      --native-consensusClient=nimbus \
-     --native-ccHttpUrl=http://localhost:{{ rocketpool_eth2_rest_port }} \
-     --native-validatorRestartCommand={{ rocketpool_service_path }}/eth2-restart.sh \
-     --native-validatorStopCommand={{ rocketpool_service_path }}/eth2-stop.sh
+     --native-ccHttpUrl={{ rocketpool_eth2_exec_layer_url }} \
```

This variable name doesn't make sense, eth2 is consensus layer, not exec layerP:
```suggestion
     --native-ccHttpUrl={{ rocketpool_eth2_rest_api_url | mandatory }} \
```

---

yes, thank you! A lot of stuff moving around so got a bit mixed up